internify/src/components/molecules/pages/FinalReqStudents.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { ChipNote } from "../../atoms/Chips";
+import "./styles/Feedback.css";
+
+const FinalReqStudents = ({ data }) => {
+  const keyArray = Object.keys(data);
+  return (
+    <div className="feedback_page4_inner">
+      <ul className="feedback_list">
+        {keyArray.map((x) => {
+          if (x === "academicReq") {
+            const msg = `${data[x]} students also meet academic requirements`;
+            return (
+              <li key={x}>
+                <ChipNote label={msg} />
+              </li>
+            );
+          } else if (x === "page3Students") {
```

Comment:
This string definition here (`"page3Students"`) is a magic number; just putting it out there that this will create tech debt. It's fine for now but please document somewhere what these definitions mean.